draft-pda-protocol.md

```diff
@@ -984,18 +966,18 @@ for s in range(0,1000,1):
 
 ### Pseudorandom number generation {#prio-prng}
```

Comment:
We might consider just punting this to the underlying CFRG document. Maybe we can move this section to the parking lot?

Reply:
Yeah, I think we're going to end up removing the Prio stuff altogether. (See #98.) But we can do so in a future PR.